org.alloytools.alloy.dash/src/main/java/ca/uwaterloo/watform/rapidDash/DashExprToPython.java

```diff
@@ -18,7 +21,7 @@ a class used to translate expressions to Python code
  */
 public class DashExprToPython<ExprType> {
     private ExprType specialExpr;
-    private StringBuilder sb;
+    private Deque<StringBuilder> sbs;
```

Comment:
Considering we may see multiple lines of expressions, we need to split them and print them on each line. Hence, the stringbuilders, which are used to generate the expression strings, are stored in lists.